### ISSUE TITLE
Large fluid extractor structure change (2.7.0 beta 3)

### ISSUE THREAD

**Issue by Atlantis102** [GTNewHorizons/GT-New-Horizons-Modpack#17905]
### Your GTNH Discord Username

atlantis102

### Your Pack Version

2.7.0 beta 3

### Your Proposal

google translate

Large fluid extractor requires a minimum of 47 robust tungstensteel machine casing out of 52 available in the structure, which greatly reduces the amount of useful space for hatches, at the moment it is even less than LPF. For such a large structure this is very little.

### Your Goal

Reduce the requirement for robust tungstensteel machine casing to at least 30 and better to 20. This way the amount of useful space in this multi-block will increase.

### Your Vision

A large number of hatches is necessary if for certain reasons you need a separate outlet for each specific liquid.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this change please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members think it's inappropriate.
- [X] I believe this feature would make the pack better.

**Comment by RecursivePineapple** [comment]
I don't think this is a problem. It can have up to 16 hatches (41 to 57 casings). With a maintenance hatch, 2 energy hatches, and 4 input busses you're left with 9 output hatches.
